Add B02 implementation

TrueFinch opened the pull request:
Add B02 tasks https://www.notion.so/EGE-B02-d2ec2250fc3a4e5c98a0fd2d102ea929

TrueFinch committed:
Add B02 implementation

https://www.notion.so/EGE-B02-d2ec2250fc3a4e5c98a0fd2d102ea929

SergeyZinkovich commented on EGE/Gen/EGE/B02.py:
```diff
@@ -0,0 +1,65 @@
+        va, vb = self.rnd.shuffle([ 'a', 'b' ])
+        loop = self.rnd.pick([ 'while', 'until' ])
+        va_init, va_end, va_op, va_arg, va_cmp = self.rnd.pick([
+            lambda: [ 0, self.rnd.in_range(5, 7), '+', 1, '<' ],
```
Есть массив лямб, которые возвращают массивы. Мы выбираем случайную лямбду, и тут же ее вызываем, получая массив. А как насчет убрать лямбды, и просто выбирать случайный массив из массива массивов?

TrueFinch replied:
Серёга — голова

SergeyZinkovich commented on EGE/Gen/EGE/B02.py:
```diff
@@ -0,0 +1,65 @@
+            lambda: [ self.rnd.in_range(15, 20), '-', self.rnd.in_range(1, 3) ],
+            lambda: [ self.rnd.in_range(1, 4), '*', self.rnd.in_range(2, 4) ],
+            lambda: [ 2 ** self.rnd.in_range(8, 10), '/', 2 ]
+        ])()
```
Ну и тут то же самое

TrueFinch committed:
Replace strange lambdas with list it returns

https://www.notion.so/EGE-B02-d2ec2250fc3a4e5c98a0fd2d102ea929 — according to https://github.com/klenin/EGEpy/pull/27#discussion_r786013972
